lib/models/samples_widget_list.dart

```diff
@@ -42,64 +43,58 @@ import 'package:arcgis_maps_sdk_flutter_samples/samples/show_portal_user_info/sh
 import 'package:arcgis_maps_sdk_flutter_samples/samples/show_service_area/show_service_area.dart';
 import 'package:arcgis_maps_sdk_flutter_samples/samples/style_point_with_simple_marker_symbol/style_point_with_simple_marker_symbol.dart';
 
-// A list of all the Widgets for individual Samples.
-// Used by the Sample Viewer App to display the Widget when a sample is selected.
-// The key is the directory name for the sample which is in snake case. E.g. display_map
-final sampleWidgets = <String, Function>{
-  'add_feature_collection_layer_from_table': () =>
+final sampleWidgets = {
+  'AddFeatureCollectionLayerFromTable': () =>
```

Comment:
This generated file is wrong, no samples are loading in the sample viewer. I'll push a fix to the sample generator.

Reply:
#125

Reply:
I will fix it and sync this file with v.next!